extensions/wikia/PortableInfobox/services/Helpers/PortableInfoboxRenderServiceHelper.php

```diff
@@ -247,7 +247,7 @@ public function getImageSizesForThumbnailer( $image ) {
 	 * is set, the $thumbnail->getWidth() can return some big value - we need
 	 * to adjust it to DESKTOP_THUMBNAIL_WIDTH to look good in the infobox.
 	 * Also, the $height have to be adjusted here to make image look good in infobox.
-	 * @param $thumbnail
+	 * @param $thumbnail \MediaTransformOutput
```

Comment:
Since we rely on the type of input object (we use its methods internally) it makes sense to me to add a hard type hint here